state/state.go

```diff
@@ -117,23 +122,84 @@ func detectPartitionByFindmnt(b *block.Partition) PartitionState {
 func detectBoot() Boot {
 	cmdline, err := os.ReadFile("/proc/cmdline")
 	if err != nil {
+		Log.Debug().Err(err).Msg("Error reading /proc/cmdline file " + err.Error())
 		return Unknown
 	}
+
 	cmdlineS := string(cmdline)
+
+	if DetectUKIboot(cmdlineS) {
```

Comment:
we actually need both sentinels, so returning here is an error.

We should have our normal sentinels (active/passive/recovery/livecd) and an extra one indicating if we are on uki or not.

With both we can then infer if we are on livecd-uki or just uki (boot from hdd) + active/passive/recovery

Reply:
Both sentinels are still created.

the uki one: https://github.com/kairos-io/immucore/pull/225/files#diff-09e12c6901252c707607e566f62f6417adda8f1a02fb1e41d5e0fec69ef7312eR393

and some lines above that, the "state" sentinel.